libs/partners/unstructured/pyproject.toml

```diff
@@ -14,7 +14,7 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 langchain-core = "^0.2.23"
-unstructured-client = { version = "^0.24.1" }
+unstructured-client = { version = ">=0.24.1" }
```

Comment:
I would keep the caret notation to prevent pulling `1.*` when it's released and that might be a non-compatible one.